litmus-portal/frontend/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Docker Image which is used as foundation to create
+# a custom Docker Image with this Dockerfile
+FROM node:12.18.0 as react-build
+
+# Add Maintainer Info
+LABEL maintainer="LitmusChaos"
+
+# A directory within the virtualized Docker environment
+# Becomes more relevant when using Docker Compose later
+RUN mkdir /frontend
```

Comment:
This line is not required, WORKDIR will cover this process

Reply:
done